roles/ocp_on_libvirt/tasks/libvirt_host_destroy.yml

```
@@ -1,20 +1,31 @@
 ---
 - name: "Does node exist"
+  community.libvirt.virt:
+    command: list_vms
+  register: _ool_vms_list
+
+- name: "Set node_not_exists"
+   ansible.builtin.set_fact:
```

Last job failed because of an extra space of indentation here.
```suggestion
  ansible.builtin.set_fact:
```